pkg/fixtures/workflows/github/helm/.github/workflows/azure-kubernetes-service-helm.yml

```diff
@@ -102,16 +104,18 @@ jobs:
 
       # Retrieves your Azure Kubernetes Service cluster's kubeconfig file
       - name: Get K8s context
-        uses: azure/aks-set-context@v3
+        uses: azure/aks-set-context@v4.0.2
         with:
           resource-group: ${{ env.CLUSTER_RESOURCE_GROUP }}
           cluster-name: ${{ env.CLUSTER_NAME }}
+          resource-type: ${{ env.CLUSTER_RESOURCE_TYPE }}
           admin: "false"
           use-kubelogin: "true"
 
       # Checks if the AKS cluster is private
       - name: Is private cluster
         id: isPrivate
+        if: ${{ env.CLUSTER_RESOURCE_TYPE != 'Microsoft.ContainerService/fleets' }}
```

Comment:
This looks case sensitive.  It may be that there's not much we can easily do about that right now.